src/Microsoft.AspNetCore.OData/Routing/ODataActionSelector.cs

```diff
@@ -95,10 +95,19 @@ public ActionDescriptor SelectBestCandidate(RouteContext context, IReadOnlyList<
         {
             RouteData routeData = context.RouteData;
             ODataPath odataPath = context.HttpContext.ODataFeature().Path;
+
             if (odataPath != null && routeData.Values.ContainsKey(ODataRouteConstants.Action))
             {
+                var odataRoute = routeData.Routers.OfType<ODataRoute>().FirstOrDefault();
```

Comment:
>routeData.Routers.OfType<ODataRoute>().FirstOrDefault(); [](start = 33, length = 56)

I'd like to use "IODataFeature" to save/retrieve the route name and route prefix. "ODataRoute" will be removed later.

Reply:
@xuzhg Can the route prefix be currently retrieved from the `IODataFeature`? Or will it be added to it in a future PR?